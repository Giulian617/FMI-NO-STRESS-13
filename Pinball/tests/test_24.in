100 90624226203273
40 53 2
6 40 8
29 53 8
33 40 5
96 6 4
4 33 4
98 53 8
61 98 5
2 4 1
85 4 1
45 33 3
5 45 9
38 53 8
31 40 5
65 45 9
1 5 4
30 6 10
72 31 4
8 40 1
60 33 10
78 65 8
75 61 1
74 61 9
91 78 1
97 5 3
41 96 9
82 8 5
43 30 3
73 72 3
55 45 7
22 8 7
26 53 1
11 26 8
35 26 7
84 60 1
34 35 8
67 85 7
27 53 2
90 73 8
48 98 5
44 73 6
47 72 7
94 82 2
87 67 3
19 78 10
86 90 5
10 98 2
3 11 5
23 43 10
79 2 3
46 19 6
66 91 5
15 38 3
93 26 8
99 45 7
39 10 3
51 66 6
13 86 7
83 85 9
16 35 4
18 30 6
49 91 1
81 74 2
28 3 5
37 38 2
92 22 2
63 65 2
52 43 9
62 34 3
14 6 10
58 46 3
12 33 9
69 13 4
36 90 2
54 99 9
71 2 4
64 65 6
50 45 3
68 71 10
80 82 4
25 65 7
17 97 5
76 90 6
7 44 7
77 15 10
88 8 9
9 49 3
70 55 3
21 73 9
20 15 3
32 17 4
95 53 1
42 76 8
89 38 8
56 19 3
59 66 8
24 45 10
57 43 5
100 70 3
