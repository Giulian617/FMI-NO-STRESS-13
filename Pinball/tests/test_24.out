305392411
755895010
816243579
19089190
856519516
436102820
53362087
668148983
2957355
53380646
268221947
645737826
807505167
127517206
225272358
550397124
865730465
311122031
215810698
861150319
781028764
80205947
265375586
727391814
848352843
892413018
154758802
520266265
286834486
672941893
584824169
39853675
974205965
152955822
885780442
611059727
522062764
242960235
300272684
829894225
678573576
682422205
920348375
197971774
191897408
988246097
927650472
127517206
570800722
340817675
154758802
